bin/check_samplesheet.py

```diff
@@ -137,14 +137,24 @@ def check_samplesheet(file_in, file_out):
 
                 ## Check that replicate ids are in format 1..<num_replicates>
                 uniq_rep_ids = sorted(list(set(sample_mapping_dict[sample].keys())))
-                if len(uniq_rep_ids) != max(uniq_rep_ids):
+                if len(uniq_rep_ids) != max(uniq_rep_ids) or 1 != min(uniq_rep_ids):
```

Comment:
👀 What does this fix?

Reply:
I think is checking whether the first replicate is not named as 1 but as any other number, then it won't fit in 1..<num_replicates>

Reply:
It prevents starting with 0 and then skipping one number, i.e 0,2,3

Reply:
Well spotted! 🚀